### ISSUE TITLE
Customization/refit time calculation errors

### ISSUE THREAD

**Issue by monbvol** [MegaMek/mekhq#981]
### Environment
0.45.1(all really)
Windows 10 64bit
Java 1.8.0_171

### Description
The time needed to customize/refit units is too long.

As a specific example I come up with 1890 minute to take a 6R Wolverine to a 6M while MekHQ says 5505 minutes.

120 minutes to replace the AC-5 with a Large Laser
120 minutes to install a new Medium Laser
90 minutes to add armor
120 minutes to remove the AC-5 ammo
180 minutes to add 2 heatsinks

630 minutes * 3 for a class D refit= 1890 minutes total.
![refittimeerror](https://user-images.githubusercontent.com/17420975/48664691-71607000-ea57-11e8-99de-3f9fd209287d.png)

Relevant rules question thread: https://bg.battletech.com/forums/index.php?topic=52669.0

### Files
As this time error occurs with any refit/customization I've tried it is not campaign specific.

**Comment by sixlettervariables** [comment]
Walking the code this is what MekHQ sees:
```
Wolverine WVR-6R (self) -> Wolverine WVR-6M (self)
[120 min] Heat Sink moved (Left Leg -> Right Torso): 120 minutes
[120 min] Medium Laser moved (Right Arm -> Head): 0 minutes
[220 min] Jump Jet moved (Left Torso -> Center Torso): 100 minutes
[320 min] Jump Jet moved (Left Torso -> Right Leg): 100 minutes
[420 min] Jump Jet moved (Right Torso -> Right Leg): 100 minutes
[520 min] Jump Jet moved (Right Torso -> Left Leg): 100 minutes
[535 min] SRM 6 Ammo Bin moved (Right Torso -> Left Torso): 15 minutes
[635 min] Jump Jet moved (Center Torso -> Left Leg): 100 minutes
[680 min] 9 Armor (Standard) armor (Head) missing 45 minutes
[805 min] 25 Armor (Standard) armor (Center Torso) missing 125 minutes
[895 min] 18 Armor (Standard) armor (Right Arm) missing 90 minutes
[985 min] 18 Armor (Standard) armor (Left Arm) missing 90 minutes
[1080 min] 19 Armor (Standard) armor (Right Leg) missing 95 minutes
[1175 min] 19 Armor (Standard) armor (Left Leg) missing 95 minutes
[1265 min] Heat Sink missing (Left Leg) 90 minutes to add
[1385 min] Large Laser missing (Right Leg) 120 minutes to add
[1475 min] Heat Sink missing (Left Torso) 90 minutes to add
[1595 min] Medium Laser missing (null) 120 minutes to add
[1595 min] Removing old AC/5 (Right Arm) 120 minutes
[1835 min] Unloading old AC/5 Ammo Bin (Right Arm) 120 minutes
[1835 min] 0 heatsinks changed, 0 minutes
Applying 3x multiplier: 1835 -> 5505 minutes
```
Looks like it may be overcharging you for armor.

**Comment by sixlettervariables** [comment]
You also got overcharged for moving a jump jet that could have stayed put.

**Comment by monbvol** [comment]
I'll admit I never noticed the 6M actually does relocate so much stuff when
it doesn't really need to but still I'm seeing multiple overcharges/moves
that don't need to be done that are driving up the time.

I'm not sure why it is moving a Jump Jet from the Center Torso when both
keep a Jump Jet there.

Also not sure why it is doing that first Heat Sink move as both have a Heat
Sink already there in the Right Torso.

Likewise don't know why it is moving the Head Medium Laser.

It is overcharging for armor.

Jump Jets are also only 60 minutes not 100 but that probably does need to
be 60x2 for pulling then putting back in.

**Comment by sixlettervariables** [comment]
I believe I addressed everything I found wrong in the algorithm itself, but some of the times may still be off. This cuts about 35% off the refit time.

**Comment by monbvol** [comment]
Yeah saw that the Medium Laser move that it doesn't need to do isn't
charged any time after I replied and asked a clarifying question on the
rules forum for if the SRM 6 should count or not and likewise if the Large
Laser should be charged separately like the code spits out.

Which I just realized I assumed the Jump Jets should be separate operations
further multiplied by the refit class modifier in my rules question when
that may not be the case.

Also noticed it is saying Right Leg in the code for the Large Laser but
fortunately it seems to actually put the Large Laser in the Right Arm.
